howso/direct/client.py

```diff
@@ -1185,7 +1185,8 @@ def train(  # noqa: C901
         series: Optional[str] = None,
         train_weights_only: bool = False,
         validate: bool = True,
-    ):
+        skip_auto_analyze: bool = False,
```

Comment:
Please move this to before `train_weights_only` as the kwargs-only args (those after the `*`) are shown here in alphabetical order since order will never matter for these.  Same for the doc-string please.